deps/rabbit/test/logging_SUITE.erl

```diff
@@ -1383,7 +1383,9 @@ log_and_return_json_object(Context, Metadata, DecodeOpts) ->
                         Content,
                         "^.+\"" ++ RandomMsg ++ "\".+$",
                         ReOpts),
-    Term = rabbit_json:decode(Line, [{labels, attempt_atom} | DecodeOpts]),
+
+    Term0 = rabbit_json:decode(Line, DecodeOpts),
+    Term = rabbit_data_coercion:atomize_keys(Term0),
```

Comment:
There was a question on the forum about decoding to existing atom keys: https://erlangforums.com/t/thoas-elixirs-jason-json-library-converted-to-erlang/879/38

Should we await some of that discussion? `Jason` has `Jason.decode(binary, keys: :atoms)` or `Jason.decode(binary, keys: :atoms!)` (existing atoms) but it looks like these may have been intentionally removed from Thoas https://github.com/lpil/thoas/commit/d672dce4aeb64a5e1c606e136238fecf453e7456

I think the separate call to `atomize_keys/1` is a good alternative though

Reply:
Yes, it is good enough for our needs